### ISSUE TITLE
Serve stale page when rendering has failed

### ISSUE THREAD

**Issue by dulnan** [dulnan/nuxt-multi-cache#55]
The route cache could have an option to serve stale pages when rendering has failed.

**Comment by dulnan** [comment]
@bennet-esyoil Thanks for your input! I agree and I've implemented it now like this. This new feature will be opt-in, using a new `setStaleIfError` method on the route cache helper:

```typescript
useRouteCache((helper) => {
  helper.setCacheable().setMaxAge(60).setStaleIfError(3600)
})
```

**Comment by dulnan** [comment]
Now available in 3.3.0
